dimelo/load_processed.py

```diff
@@ -14,16 +14,28 @@ def pileup_counts_from_bedmethyl(
     bedmethyl_file: str | Path,
     motif: str,
     regions: str | Path | list[str | Path] | None = None,
+    # window_size: int | None = None,
+    single_strand: bool = False,
 ) -> tuple[int, int]:
     """
-    Extract number of modified bases and total number of bases from the given bedmethyl file
+    Extract number of modified bases and total number of bases from the given bedmethyl file.
+    Called by plotters or by the user.
 
-    TODO: How to name this method?
+    This function loops through all the provided regions and pulls those regions up in your
+    sorted and indexed bedmethyl file. For rows within those regions, we check that the motif
+    is correct (i.e. sequence context, modified base, mod code, and optionally strand). All
+    correct locations are included in the sum counts that get returned.
+
+    If no regions are specified, we return the sum total for the motif of interest across the
+    entire bedmethyl file.
```

Comment:
Nitpick: I think this wording could be improved. Also we need to establish consistency in how we address the user / ourselves in doc strings. I usually try to avoid using first or second person in doc strings to be more formal, but this really isn't that important. In the future I won't add these nitpick things unless it's necessary but for the first PR review I'm just noting stuff for the future I guess.